src/pr-info.ts

```diff
@@ -363,7 +363,7 @@ const configSuspicious = <ConfigSuspicious>(async (path, getContents) => {
     const basename = path.replace(/.*\//, "");
     if (!(basename in configSuspicious)) return `edited`;
     const text = await getContents();
-    if (text === undefined) return `couldn't fetch contents`;
+    if (text === undefined) return undefined;
```

Comment:
This looks fine, but the justification from the PR description is important in case more json files are added in the future.  So can you please add a comment before this line with a quick summary of it?  (Possibly squash + reword the commits to add the verbose explanation to the log message, then in the short comment refer to the log message.)

Reply:
Done :heavy_check_mark: